tests/contrib/test_minipyro.py

```diff
@@ -186,18 +201,47 @@ def guide():
 def test_constraints(backend, jit):
     data = torch.tensor(0.5)
 
-    def model():
-        locs = pyro.param("locs", torch.randn(3), constraint=constraints.real)
-        scales = pyro.param("scales", ops.exp(torch.randn(3)), constraint=constraints.positive)
-        p = torch.tensor([0.5, 0.3, 0.2])
-        x = pyro.sample("x", dist.Categorical(p))
-        pyro.sample("obs", dist.Normal(locs[x], scales[x]), obs=data)
+    with pyro_backend(backend):
+        Elbo = infer.JitTrace_ELBO if jit else infer.Trace_ELBO
+        elbo = Elbo(ignore_jit_warnings=True)
+        assert_ok(constrained_model, guide_constrained_model, elbo, 2, data)
 
-    def guide():
-        q = pyro.param("q", ops.exp(torch.randn(3)), constraint=constraints.simplex)
-        pyro.sample("x", dist.Categorical(q))
 
+@pytest.mark.parametrize("backend", ["pyro", "minipyro"])
+def test_elbo_jit(backend):
+    """
+    Given model and guide
+    After 15 iterations of
+    SVI, JitTrace_ELBO yield
+    similar results for both pyro and minipyro.
+    This is a relaxed assertion over
+    the one reached in test_elbo_equivalence
+    Note: 15 is the minimun number for iterations for this
+    to happen.
+    """
+    pyro.set_rng_seed(0)
+    data = torch.tensor(0.5)
+    elbo_test_case(backend, True, 0.4780, data, 15)
+
+
+@pytest.mark.parametrize(["backend", "jit"], [("pyro", True), ("pyro", False), ("minipyro", False)])
+def test_elbo_equivalence(backend, jit):
+    """
+    Given model and guide
+    Calculating the elbo yields equivalent results
+    """
+    pyro.set_rng_seed(0)
+    data = torch.tensor(0.5)
+    elbo_test_case(backend, jit, expected_elbo=0.4780, data=data)
```

Comment:
ditto

Reply:
Apologies I don't follow what's the suggestion in this case